assets/javascripts/kitten/components/tooltips/static-tooltip.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+export class StaticTooltip extends React.Component {
+  renderArrow() {
+    let style
+
+    if (this.props.arrowLeftPosition) {
```

Comment:
Tu pourras très bien les rajouter en fonction de `this.props. direction`. C'est beaucoup plus maintenable que de devoir créer 4 props pour chaque direction.